test/rest-openapi-sink/openapi.json

```diff
@@ -0,0 +1,289 @@
+{
```

Comment:
Can you add license?

Reply:
Standard Json does not support comments so it is hard to add the license here. Have added the license headers to other resources though. Thanks for spotting